src/python/pants/engine/internals/graph.py

```diff
@@ -261,7 +261,9 @@ async def addresses_with_origins_from_filesystem_specs(
             msg = (
                 f"No owning targets could be found for the file `{file_path}`.\n\nPlease check "
                 f"that there is a BUILD file in `{file_path.parent}` with a target whose `sources` field "
-                f"includes `{file_path}`. See https://www.pantsbuild.org/build_files.html."
+                f"includes `{file_path}`. See https://www.pantsbuild.org/build_files.html for more "
+                "information on target definitions.\n"
```

Comment:
I've found two new lines makes error messages even more readable.

Reply:
This error message already contains a two line break between the actual error and these two help blurbs: double breaking the help blurbs would look odd I think.

Reply:
Fair.